rails-task-list/app/views/tasks/new.html.erb

```diff
@@ -1,7 +1,12 @@
-<h1> Add a new task </h1>
+<% title = params[:action] %>
```

Comment:
Preferably your view logic inside of your templates should never be checking `params` directly. Instead, they should only make use of instance variables provided for them by the controller. This way, if we ever decide to change what's in `params` then only the controller will need to be updated.